lib/ansible/utils/color.py

```diff
@@ -85,12 +85,15 @@ def parsecolor(color):
         return u'38;5;%d' % (232 + int(matches.group('gray')))
 
 
-def stringc(text, color):
+def stringc(text, color, for_prompt=False):
     """String in color."""
 
     if ANSIBLE_COLOR:
         color_code = parsecolor(color)
-        return u"\n".join([u"\033[%sm%s\033[0m" % (color_code, t) for t in text.split(u'\n')])
+        fmt = u"\033[%sm%s\033[0m"
+        if for_prompt:
```

Comment:
Maybe add a comment here from the readline docs explaining what this is doing so we will know in the future since this it's not obvious from the code.
```suggestion
        if wrap_nonvisible_chars:
```

Reply:
done